resources/prefs.json

```diff
@@ -1,7 +1,7 @@
 {
   "dom.bluetooth.enabled": false,
   "dom.bluetooth.testing.enabled": false,
-  "dom.canvas-text.enabled": false,
+  "dom.canvas-text.enabled": true,
```

Comment:
Sounds good! I thought the goal here was to prevent the JS Exception by default.